os/include/tinyara/kmalloc.h

```diff
@@ -144,9 +149,17 @@ extern "C" {
 #define kmm_givesemaphore(a)     umm_givesemaphore(a)
 
 #if CONFIG_MM_NHEAPS > 1 && CONFIG_MM_REGIONS > 1
+
+#ifdef CONFIG_FS_TMPFS_HEAP_INDEX
 #define kmm_malloc(s)          malloc_at(CONFIG_FS_TMPFS_HEAP_INDEX, s)
```

Comment:
this config naming seems related to filesystem tmpfs, 
why  malloc policy is related to FS?

Reply:
that is previous logic for tmpfs to use multiheap, and that is rely on the config index. so we just add a new condition for no FS config case as default